azure-setup-guide/terraform/modules/kafka_app/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "kafka_bootstrap_server" {
```

Comment:
again, is this terraform standard to have the outputs, variables, etc in different files? Elm do it this way, re-frame did it too, and I always found it annoying because it broke the 'locality of related things' principle - one logical change unnecessarily means changing/switching back-n-forth around multiple files.

Reply:
From what I read on the terraform docs, it does seem somewhat like the standard yeah: https://developer.hashicorp.com/terraform/tutorials/modules/module-create